ttnn/cpp/pybind11/pytensor.cpp

```diff
@@ -66,17 +66,17 @@ void log_external_operation(
 #endif
 
 template <typename T>
-Tensor create_owned_tensor(
-    T* data_ptr,
-    size_t num_elements,
-    tt::stl::Span<const uint32_t> shape,
-    DataType data_type,
-    Layout layout,
-    const std::optional<Tile>& optional_tile = std::nullopt) {
-    auto data = std::vector(data_ptr, data_ptr + num_elements);
+Tensor create_owned_tensor(T* data_ptr, const ttnn::TensorSpec& tensor_spec) {
```

Comment:
data_ptr here is raw unprepared data. can't be data taken from other tensor's ownedstorage's buffer

true?

Reply:
yes

Reply:
I intend this function to be generalized into robust helper, but the interface into it needs to be reworked.